easybuild/easyconfigs/p/PGI/PGI-16.4-GCC-5.3.0-2.26.eb

```diff
@@ -0,0 +1,25 @@
+name = 'PGI'
+version = '16.4'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86_64.tar.gz']
+checksums = ['3804dc13b8cf3d50aa16fb56e2682dd7']
+
+gccver = '5.3.0'
+binutilsver = '2.26'
+versionsuffix = '-GCC-%s-%s' % (gccver, binutilsver)
+
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '', ('GCCcore', gccver)),
+    ('numactl', '2.0.11', '', ('GCCcore', gccver)),
+]
+
```

Comment:
since all of these are installed by default by the easyblock, I see no reason to highlight that they're there, just feel redundant...

for `install_java`, we should add detection in the easyblock that `Java` is listed as a dependency, and automagically switch to `install_java = False`?

anyway, this should be discussed in a separate PR/issue imho, I'm not a big fan of including our own Java here, partially because the Java we install is a full JDK (development kit), while PGI only includes the JRE (runtime environment), potentially with additional stuff slapped onto it to make the graphical debugger work...